srcpkgs/vulkan-headers/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'vulkan-headers'
+pkgname=vulkan-headers
+_pkgname=Vulkan-Headers
```

Comment:
why not just call the package itself  Vulkan-Headers ?

Reply:
The loader package is already called vulkan-loader, and I wanted to keep things consistent. If we're renaming the headers package, I would like to rename the loader package as well. I imagine this could cause some problems, though.